### ISSUE TITLE
`SPC b b` in a layout displays irrelevant buffers

### ISSUE THREAD

**Issue by deb0ch** [syl20bnr/spacemacs#7097]
#### Description :octocat:
`SPC b b` in a layout displays all (or not ?) buffers instead of restricting to current layout buffers.

#### Reproduction guide :beetle:
- Start Emacs
- create a layout
- add buffers to it
- `SPC b b`

*Observed behaviour:* :eyes: :broken_heart:
All buffers are listed.
I am not sure, but I might have noticed once that some of the buffers present in the layout were not listed.

*Expected behaviour:* :heart: :smile:
Only buffers present in the current layout should be listed.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.22
- Spacemacs branch: perso (rev. c4f8cb4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
((auto-completion :variables :disabled-for markdown org erc)
 (better-defaults :variables better-defaults-move-to-beginning-of-code-first nil better-defaults-move-to-end-of-code-first t)
 c-c  
 (colors :variables colors-enable-nyan-cat-progress-bar nil)
 common-lisp csv emacs-lisp git github gtags helm markdown multiple-cursors org pdf-tools python ranger
 (semantic :disabled-for c  -mode)
 (shell :variables shell-default-height 50 shell-default-position 'bottom)
 shell-scripts spacemacs-layouts syntax-checking version-control vim-empty-lines ycmd)
```

**Comment by TheBB** [comment]
Probably fixed now? https://github.com/syl20bnr/spacemacs/pull/7142

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
